Felienne/hedy
# Fix typos in en.yaml

Changes the "en"s (Dutch for "and") to "and"s, and renames "mension" to "mansion" in `en.yaml`

**How to test**

This is a language change. Run Hedy and then go to http://localhost:8080/hedy/9#haunted and http://localhost:8080/hedy/16#haunted to see the difference.

lvl 9 haunted house new: (note the and/en at the end of the sentence)
<img width="592" alt="9new" src="https://user-images.githubusercontent.com/5196627/164461442-622e56aa-cab1-43f1-9f67-752639430e15.png">

lvl 9 haunted house old:
<img width="581" alt="9old" src="https://user-images.githubusercontent.com/5196627/164461535-c111299d-6b55-4ee8-a57b-edcc486819ff.png">

lvl 16 haunted house new (also note mension->mansion near the top right:
<img width="570" alt="16new" src="https://user-images.githubusercontent.com/5196627/164462176-673d8f67-9d17-459f-b311-95b40b4af195.png">

lvl 16 haunted house old:
<img width="570" alt="16old" src="https://user-images.githubusercontent.com/5196627/164461611-99bfe4a7-4c92-4183-a217-c96bb6c9b5f3.png">


This PR doesn't reference an issue attached as I forgot to create one and I think this is a really small change. Feel free to reject the PR if there is some good reason to create an issue first.
